### DISCUSSION TITLE
How does update work after --year?

### DISCUSSION THREAD

**Discussion by swduncan** [RhetTbull/osxphotos#986] [Q&A] [answered]
I'm exporting a large (85k+ images, 720+gb) library, and I've been doing it by year using --year to keep the exports to an hour or so. If I just use update and remove the --year XXXX will it export the remainder of the library?

**Comment by RhetTbull** [comment, marked as answer]
Yes, if you use --update after dropping a filter (for example --year), any remaining photos should be exported (as well as any changes to previously exported photos). Ideally, you should have been using --update all along with the --year filter to prevent any name collisions. If your directory structure is such that there would be no collisions then this shouldn't matter. You can always use --dry-run --verbose to see what will be updated. You can also use --limit X with --update to limit the export to X photos at a time.

**Reply by swduncan** [reply]
Perfect! Thanks!
